testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/CamundaProcessTestExtension.java

```diff
@@ -263,11 +263,26 @@ private static boolean isTestFailed(final ExtensionContext extensionContext) {
    *
    * @param camundaVersion the version to use
    * @return the extension builder
+   * @deprecated use withCamundaDockerImageVersion instead.
+   * @since 8.8.0
    */
+  @Deprecated
   public CamundaProcessTestExtension withCamundaVersion(final String camundaVersion) {
+    return withCamundaDockerImageVersion(camundaVersion);
+  }
+
+  /**
+   * Configure the Camunda docker image version of the runtime.
+   *
+   * @param camundaDockerImageVersion the version to use
+   * @return the extension builder
+   */
+  public CamundaProcessTestExtension withCamundaDockerImageVersion(
+      final String camundaDockerImageVersion) {
+
     runtimeBuilder
-        .withCamundaDockerImageVersion(camundaVersion)
-        .withConnectorsDockerImageVersion(camundaVersion);
+        .withCamundaDockerImageVersion(camundaDockerImageVersion)
+        .withConnectorsDockerImageVersion(camundaDockerImageVersion);
```

Comment:
:x: The method should set only the version of the Camunda image. The previous method set both versions together.